source/MRViewer/ImGuiMenu.cpp

```diff
@@ -602,6 +602,8 @@ void ImGuiMenu::draw_helpers()
         for ( int i = 0; i<int( Viewer::EventType::Count ); ++i )
             ImGui::Text( "%s: %zu", cEventCounterNames[i], viewer->getEventsCount( Viewer::EventType( i ) ) );
         ImGui::Separator();
+        auto glBufferSizeStr = bytesString( viewer->getStaticGLBufferSize() );
+        ImGui::Text( "GL memory buffer: %s", glBufferSizeStr.c_str() );
```

Comment:
Why not use GLStaticBuffer directly?

Reply:
GLStaticBuffer is internal class used for rendering purposes only, there's no reason to have full access to it from other classes.